pkg/option/config.go

```diff
@@ -146,6 +146,13 @@ const (
 
 	// SockopsEnableName is the name of the option to enable sockops
 	SockopsEnableName = "sockops-enable"
+
+	// K8sNamespaceName is the name of the K8sNamespace option
+	K8sNamespaceName = "k8s-namespace"
+
+	// K8sNamespaceNameEnv is the name of the K8sNamespace environment
+	// variable
+	K8sNamespaceNameEnv = "CILIUM_NAMESPACE"
```

Comment:
I've some local changes that will make the environment variables to be consistent with the options set in cilium-agent, to help with that can you change this option to `CILIUM_K8S_NAMESPACE` instead?

Reply:
Sure

Reply:
Sure